r4/src/main/java/org/opencds/cqf/r4/providers/PlanDefinitionApplyProvider.java

```diff
@@ -8,19 +8,9 @@
 
 import javax.xml.bind.JAXBException;
 
+import org.fhir.ucum.Canonical;
 import org.hl7.fhir.exceptions.FHIRException;
-import org.hl7.fhir.r4.model.ActivityDefinition;
-import org.hl7.fhir.r4.model.CarePlan;
-import org.hl7.fhir.r4.model.DomainResource;
-import org.hl7.fhir.r4.model.Extension;
-import org.hl7.fhir.r4.model.IdType;
-import org.hl7.fhir.r4.model.Parameters;
-import org.hl7.fhir.r4.model.PlanDefinition;
-import org.hl7.fhir.r4.model.Reference;
-import org.hl7.fhir.r4.model.RelatedArtifact;
-import org.hl7.fhir.r4.model.RequestGroup;
-import org.hl7.fhir.r4.model.Resource;
-import org.hl7.fhir.r4.model.StringType;
+import org.hl7.fhir.r4.model.*;
```

Comment:
Personally I'm not a fan of the wildcard import because it makes it hard to track where a class came from and can result in name clashes.

Eventually we should add checkstyle or another modern linter to handle this.